samples/excel/89-preview-apis/pivottable-calculations.yaml

```diff
@@ -0,0 +1,217 @@
+id: excel-pivottable-calculations
+name: PivotTable Calculations
+description: Change the base calculations the PivotTable is performing.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.8
+script:
+    content: |-
+        $("#setup").click(() => tryCatch(setup));
+        $("#deletePivot").click(() => tryCatch(deletePivot));
+        $("#filter").click(() => tryCatch(filter));
+        $("#getCrateTotal").click(() => tryCatch(getCrateTotal));
+        $("#minFunc").click(() => tryCatch(minFunc));
+        $("#maxFunc").click(() => tryCatch(maxFunc));
```

Comment:
This line should be removed, as the function `maxFunc` does not exist in this snippet.